test/functional/mempool_packages.py

```diff
@@ -90,9 +90,18 @@ def run_test(self):
             self.log.info("too-long-ancestor-chain successfully rejected")
 
         # Check that prioritising a tx before it's added to the mempool works
+        # First clear the mempool by mining a block.
         self.nodes[0].generate(1)
+        sync_blocks(self.nodes)
```

Comment:
need to update this, it was moved inside the framework recently (#2254)
```suggestion
        self.sync_blocks()
```